connect/runtime/src/test/java/org/apache/kafka/connect/integration/RebalanceSourceConnectorsIntegrationTest.java

```diff
@@ -157,7 +160,8 @@ public void testReconfigConnector() throws Exception {
                 CONNECTOR_SETUP_DURATION_MS, "Connector tasks did not start in time.");
 
         int numRecordsProduced = 100;
-        long recordTransferDurationMs = TimeUnit.SECONDS.toMillis(30);
+        long recordWriteDurationMs = TimeUnit.SECONDS.toMillis(60);
+        long recordTransferDurationMs = TimeUnit.SECONDS.toMillis(60);
```

Comment:
Part 2: Increased the duration for consuming messages produced by the connector, but *added* another check on the connector to write the records after it was started (see below).

Reply:
BTW, this effectively increases to 120 seconds (from 30 seconds) the timeout for waiting until the consumer can read the expected # of records. The KAFKA issue reported a bunch of timeout errors at this point as well.